webapp/src/app.tsx

```diff
@@ -1118,11 +1118,12 @@ export class ProjectView
                 pkg.mainEditorPkg().setSimState(k, v)
             },
             editor: this.state.header ? this.state.header.editor : ''
-        })
-        this.forceUpdate(); // we now have editors prepared
-
-        // start blockly load
-        this.loadBlocklyAsync();
+        }).then(() => {
```

Comment:
Can we make this use async/await instead?

Reply:
This is happening in React's `componentDidMount` callback and so I assumed it didn't support async, but some research reveals otherwise! I will try it out.